modules/atomic-widgets/prop-types/background-image-overlay-prop-type.php

```diff
@@ -13,11 +13,13 @@ public static function get_key(): string {
 	protected function define_shape(): array {
 		return [
 			'image' => Image_Prop_Type::make(),
-			'position' => String_Prop_Type::make()->enum( self::get_position_enum_values() ),
 			'repeat' => String_Prop_Type::make()->enum( [ 'repeat', 'repeat-x', 'repeat-y', 'no-repeat' ] ),
 			'size' => Union_Prop_Type::make()
 				->add_prop_type( String_Prop_Type::make()->enum( [ 'auto', 'cover', 'contain' ] ) )
 				->add_prop_type( Background_Image_Overlay_Size_Scale_Prop_Type::make() ),
+			'position' => Union_Prop_Type::make()
```

Comment:
Maybe revert it to the original position, so that we can see the difference more easily?

Or is there a specific reason why the order of the props has changed?